src/connectors/hierarchical-menu/connectHierarchicalMenu.js

```diff
@@ -296,3 +274,19 @@ export default function connectHierarchicalMenu(renderFn, unmountFn = noop) {
     };
   };
 }
+
```

Comment:
should https://github.com/algolia/instantsearch.js/pull/4204/files#diff-6ea8d65722bc8ade374d1b33d23d90caR252 not use `getCurrentLimit()` too? 🤔

Reply:
No because `showMore` is not passed to the enhancer. It can be used if we address https://github.com/algolia/instantsearch.js/pull/4204#discussion_r350227854 and that we add `getFacetsLimit()` on top of `getCurrentLimit()`.

- `getCurrentLimit()` would get the UI limit
- `getFacetsLimit()` would get the search parameter computed based on whether `showMore` is provided

Reply:
That seems to be ideal to me

Reply:
Done in a4ccbb2ba43233b08b0b126a8ae4d51377e83b8e.